content/doc/developer/testing/index.adoc

```diff
@@ -52,19 +52,34 @@ For example:
 You are encouraged to test that your plugin works with link:/doc/pipeline/[Pipeline].
 You can do so without making your plugin itself dependent on various Pipeline-related plugins; instead you can include these as `test` dependencies, so that they are only used when compiling and running test cases.
 
-The simplest way to do this is to add plugin:workflow-aggregator[`workflow-aggregator`], which is the "parent" Pipeline plugin, to the `<dependencies>` section of your POM, like this:
+This is done by adding critical Pipeline plugins to the `<dependencies>` section of your POM.
```

Comment:
It seems to me that 'critical' is not quite the right word for this condition.  If you're feeling that we need to clarify the subset of Pipeline plugins that are used, then would you consider "necessary" or "required" rather than "critical"?
```suggestion
This is done by adding Pipeline plugins to the `<dependencies>` section of your POM.
```